server/models/delius/deliusStaffDetails.ts

```diff
@@ -0,0 +1,9 @@
+export interface DeliusStaffDetails {
```

Comment:
Do any of the fields on these interfaces need to be optional? One thing I've learned when working with the community api is that, particularly locally, all of these fields can be missing from the response, but that might not be the case here.

Reply:
`username` should never be null given that it's used for the query.
`startDate` should never be null looking at the communityApi code it defaults the value to the current date. I'll confirm this with Rajiv if he's able to query the database.
`teams` is defaulted to an empty list in communityApi if no teams exist and can never be null.

`emailAddress` and `telephone` are nullable - but making them optional shouldn't change anything (I'll do that)

Reply:
You were right, I've just tested with real communityApi and teams can be nullable (not sure how since the code should default it to an empty list).
I'll make that change

Reply:
I've now added in these changes.